roles/database/tasks/main.yml

```diff
@@ -0,0 +1,12 @@
+---
+- include_role:
+    name: vendor/geerlingguy.mysql
+  vars:
+    mysql_databases:
+      - name: donalo_staging
```

Comment:
Can this be inferred from host? Like `donalo_production`, `donalo_staging`, etc.